test/fuzz/read_fuzzer.cc

```diff
@@ -0,0 +1,23 @@
+#include <pcl/io/ply_io.h>
```

Comment:
I think we should have a test for this in PCL at least (compilation test at a minimum). It'll not be good if we break something accidentally and our CI remains green.

Thoughts?

Reply:
If changes in ply_io.h cause the fuzzer to be unable to run, PCL will not be affected. The fuzzer is independent of PCL, and in case it will not be able to run, you will simply get a notification from OSS-fuzz that the fuzzer could not be executed, and we can work towards a fix. PCL can and will work perfectly well if the fuzzer is broken.